src/Cech_complex/include/gudhi/Cech_complex_blocker.h

```diff
@@ -102,9 +102,6 @@ class Cech_blocker {
         if (kernel_.squared_distance_d_object()(sph.first, cc_ptr_->get_point(face_opposite_vertex.second)) <= sph.second) {
             is_min_enclos_ball = true;
             sc_ptr_->assign_key(sh, sph_key);
-#if CGAL_VERSION_NR >= 1050000000
-            if(exact_) CGAL::exact(sph.second);
```

Comment:
I just realized after reading #666 that this is useless now since we get the filtration from the face.